includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2831,6 +2831,19 @@ public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $f
 		];
 	}
 
+	/**
+	 * Returns the list of enabled payment method types that will function with the current checkout filtered by fees.
+	 *
+	 * @param string $order_id optional Order ID.
+	 * @param bool   $force_currency_check optional Whether the currency check is required even if is_admin().
+	 * @return string[]
+	 */
+	public function get_payment_method_ids_enabled_at_checkout_filtered_by_fees( $order_id = null, $force_currency_check = false ) {
+		return [
```

Comment:
Can we replace it with `return $this->get_payment_method_ids_enabled_at_checkout( $order_id, $force_currency_check )`? Tthis way, we maintain a relation to the source of payment methods.

Reply:
Agree with the above comment.

Reply:
Never mind. I see that the UPE gateway inherits from the WCPay gateway class.